pset-functions.js

```diff
@@ -76,6 +91,31 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function myScore (word){
+  const letters = word.toUpperCase().split('')
+  let sum = 0
+  letters.forEach((letter) => {
+    if (["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"].includes(letter)){
```

Comment:
Really great use of the forEach method coupled with if/else blocks and the .includes method for your conditional; very well done!